src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/TextBoxTests/TextBoxTests.cs

```diff
@@ -323,7 +323,7 @@ public void TextBox_Formatting_FlickerText()
 			using (new AssertionScope())
 			{
 				text2.Should().Be(text1, because: "Text content should not change at max length.");
-				text3.Should().Be("TextChanged: 1");
+				text3.Should().Be("TextChanged: 2");
```

Comment:
What's different in this context?

Reply:
During a "typing sequence" we previously only fired TextChanged once at the end. To match WinUI (and to fix an autosuggestbox-related problem), we now fire it each time. In this test, the text is set but then set again by the IInputFilter, so previously a single TextChanged was fired at the end, now it gets fired twice.